Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -142,12 +143,16 @@ - (void)titleBarButtonPressed {
 
 #pragma mark - Accessors
 
-- (void)setRefreshControl:(UIRefreshControl *)refreshControl {
-    [_refreshControl removeFromSuperview];
-
-    _refreshControl = refreshControl;
+- (UIRefreshControl *)refreshControl {
+    WMFAssertMainThread(@"Refresh control can only be accessed from the main thread");
```

Comment:
When the feed was refreshed before the view loaded, the refresh control wouldn't be shown. This fixes that issue by ensuring the refresh control & view load in those cases.